feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/autodiscovery/AccountAutoDiscoveryContent.kt

```diff
@@ -1,80 +1,187 @@
 import app.k9mail.feature.account.setup.R
 import app.k9mail.feature.account.setup.ui.autodiscovery.AccountAutoDiscoveryContract.Event
 import app.k9mail.feature.account.setup.ui.autodiscovery.AccountAutoDiscoveryContract.State
-import app.k9mail.feature.account.setup.ui.autodiscovery.item.contentItems
+import app.k9mail.feature.account.setup.ui.autodiscovery.view.AutoDiscoveryResultApprovalView
+import app.k9mail.feature.account.setup.ui.autodiscovery.view.AutoDiscoveryResultView
 
 @Composable
 internal fun AccountAutoDiscoveryContent(
     state: State,
     onEvent: (Event) -> Unit,
     oAuthViewModel: AccountOAuthContract.ViewModel,
-    contentPadding: PaddingValues,
     modifier: Modifier = Modifier,
 ) {
-    ResponsiveWidthContainer(
+    Background(
         modifier = Modifier
+            .fillMaxSize()
             .testTag("AccountAutoDiscoveryContent")
-            .padding(contentPadding)
-            .fillMaxWidth()
             .then(modifier),
     ) {
-        val resources = LocalContext.current.resources
+        val scrollState = rememberScrollState()
 
-        ContentLoadingErrorView(
-            state = rememberContentLoadingErrorViewState(state),
-            loading = {
-                LoadingView(
-                    message = stringResource(id = R.string.account_setup_auto_discovery_loading_message),
-                    modifier = Modifier.fillMaxSize(),
-                )
-            },
-            error = {
-                ErrorView(
-                    title = stringResource(id = R.string.account_setup_auto_discovery_loading_error),
-                    message = state.error?.toResourceString(resources),
-                    onRetry = { onEvent(Event.OnRetryClicked) },
-                    modifier = Modifier.fillMaxSize(),
-                )
-            },
-        ) {
-            LazyColumn(
-                modifier = Modifier
-                    .fillMaxSize()
-                    .imePadding(),
-                verticalArrangement = Arrangement.spacedBy(MainTheme.spacings.double, Alignment.CenterVertically),
+        ResponsiveWidthContainer {
+            Column(
+                modifier = Modifier.fillMaxSize(),
             ) {
-                contentItems(
-                    state = state,
-                    onEvent = onEvent,
-                    oAuthViewModel = oAuthViewModel,
+                Column(
+                    modifier = Modifier
+                        .weight(1f)
+                        .verticalScroll(scrollState)
+                        .imePadding(),
+                ) {
+                    AppTitleTopHeader(
+                        title = stringResource(id = R.string.account_setup_title),
+                        modifier = Modifier
+                            .padding(
+                                start = MainTheme.spacings.double,
+                                end = MainTheme.spacings.double,
+                            ),
```

Comment:
This is adding additional horizontal padding and leads to the header text now being two lines on my Pixel 2 when it previously was only one.

Reply:
There are sadly some variations of the header in the design. I think they need to be streamlined to do this properly.

Reply:
Also the header breaks as soon the text is slightly longer